web/client/actions/filterHistory.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+
+const INIT_FILTER_HISTORY = 'FILTER_HISTORY:INIT_FILTER_HISTORY';
+const APPLIED_FILTER = 'FILTER_HISTORY:APPLIED_FILTER';
+const STORE_CURRENT_APPLIED_FILTER = 'FILTER_HISTORY:STORE_CURRENT_APPLIED_FILTER';
+const RESTORE_CURRENT_SAVED_FILTER = 'FILTER_HISTORY:RESTORE_CURRENT_SAVED_FILTER';
+const APPLY_FILTER = 'FILTER_HISTORY:APPLY_FILTER';
+const OPEN_QUERY_BUILDER = 'LAYER_FILTER:OPEN_QUERY_BUILDER';
```

Comment:
These actions could be part of the queryform reducer and actions. FilterHistory is anyway a bad name. See my comment about history, anyway

Reply:
I prefer to keep separate, queryform reducer/actions are already enough long files full of actions. 
As suggested I have modified the name.